packages/ai-ollama/src/node/ollama-language-model.ts

```diff
@@ -77,12 +88,14 @@ export class OllamaModel implements LanguageModel {
     }
 
     protected async handleStructuredOutputRequest(ollama: Ollama, request: LanguageModelRequest): Promise<LanguageModelParsedResponse> {
+        const settings = this.getSettings(request);
```

Comment:
This merges the other way around, the speficic request settings will be overwritten by the default settings